### ISSUE TITLE
Errors installing application

### ISSUE THREAD

**Issue by adieyal** [civic-commons/opencivic#114]
Hi

I'm struggling to install the application. While running the "Prepare Site" component and choosing the modules that I want to install, I get the following error and I'm not sure how to continue:

Notice: Trying to get property of non-object in _fe_block_get_bid() (line 576 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/features_extra/fe_block.module).
Warning: array_search() expects parameter 2 to be array, null given in facetapi_get_block_info() (line 44 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/facetapi/facetapi.block.inc).
Warning: array_search() expects parameter 2 to be array, null given in facetapi_get_block_info() (line 44 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/facetapi/facetapi.block.inc).
Warning: array_search() expects parameter 2 to be array, null given in facetapi_get_block_info() (line 44 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/facetapi/facetapi.block.inc).
Warning: array_search() expects parameter 2 to be array, null given in facetapi_get_block_info() (line 44 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/facetapi/facetapi.block.inc).
Warning: array_search() expects parameter 2 to be array, null given in facetapi_get_block_info() (line 44 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/facetapi/facetapi.block.inc).
Warning: array_search() expects parameter 2 to be array, null given in facetapi_get_block_info() (line 44 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/facetapi/facetapi.block.inc).
Warning: array_search() expects parameter 2 to be array, null given in facetapi_get_block_info() (line 44 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/facetapi/facetapi.block.inc).
Warning: array_search() expects parameter 2 to be array, null given in facetapi_get_block_info() (line 44 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/facetapi/facetapi.block.inc).
Warning: array_search() expects parameter 2 to be array, null given in facetapi_get_block_info() (line 44 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/facetapi/facetapi.block.inc).
Warning: array_search() expects parameter 2 to be array, null given in facetapi_get_block_info() (line 44 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/facetapi/facetapi.block.inc).
Warning: array_search() expects parameter 2 to be array, null given in facetapi_get_block_info() (line 44 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/facetapi/facetapi.block.inc).
Warning: array_search() expects parameter 2 to be array, null given in facetapi_get_block_info() (line 44 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/facetapi/facetapi.block.inc).
Warning: array_search() expects parameter 2 to be array, null given in facetapi_get_block_info() (line 44 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/facetapi/facetapi.block.inc).
Warning: array_search() expects parameter 2 to be array, null given in facetapi_get_block_info() (line 44 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/facetapi/facetapi.block.inc).
Warning: array_search() expects parameter 2 to be array, null given in facetapi_get_block_info() (line 44 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/facetapi/facetapi.block.inc).
Warning: array_search() expects parameter 2 to be array, null given in facetapi_get_block_info() (line 44 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/facetapi/facetapi.block.inc).
Warning: array_search() expects parameter 2 to be array, null given in facetapi_get_block_info() (line 44 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/facetapi/facetapi.block.inc).
Warning: array_search() expects parameter 2 to be array, null given in facetapi_get_block_info() (line 44 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/facetapi/facetapi.block.inc).
Notice: Array to string conversion in EntityAPIControllerExportable->applyConditions() (line 624 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/entity/includes/entity.controller.inc).
Notice: Array to string conversion in EntityAPIControllerExportable->applyConditions() (line 624 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/entity/includes/entity.controller.inc).
Notice: Array to string conversion in EntityAPIControllerExportable->applyConditions() (line 624 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/entity/includes/entity.controller.inc).
Notice: Array to string conversion in EntityAPIControllerExportable->applyConditions() (line 624 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/entity/includes/entity.controller.inc).
Notice: Array to string conversion in EntityAPIControllerExportable->applyConditions() (line 624 of /home/adi/Development/civiccommons/app2/profiles/opencivic/modules/contrib/entity/includes/entity.controller.inc).
PDOException: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'better_exposed_filters' for key 'PRIMARY': INSERT INTO {l10n_update_project} (name, project_type, core, version, l10n_server, l10n_path, status) VALUES (:db_insert_placeholder_0, :db_insert_placeholder_1, :db_insert_placeholder_2, :db_insert_placeholder_3, :db_insert_placeholder_4, :db_insert_placeholder_5, :db_insert_placeholder_6); Array ( [:db_insert_placeholder_0] => better_exposed_filters [:db_insert_placeholder_1] => module [:db_insert_placeholder_2] => 7.x [:db_insert_placeholder_3] => 7.x-3.0-beta3 [:db_insert_placeholder_4] => [:db_insert_placeholder_5] => http://ftp.drupal.org/files/translations/7.x/better_exposed_filters/better_exposed_filters-7.x-3.0-beta3.%language.po [:db_insert_placeholder_6] => 1 ) in drupal_write_record() (line 7136 of /home/adi/Development/civiccommons/app2/includes/common.inc).

**Comment by pmackay** [comment]
Can you describe what order you installed the components in? It can help to install one by one.

Sheldon is off this week, he should be able to look at this more next week. I dont have much time this week either as europecommons.org is launching in next few days, but also could take a closer look soon.

**Comment by adieyal** [comment]
Thanks. I'll try again. Should I install from the top down?

Adi

On 26 June 2013 14:06, Paul Mackay notifications@github.com wrote:

> Can you describe what order you installed the components in? It can help
> to install one by one.
> 
> Sheldon is off this week, he should be able to look at this more next
> week. I dont have much time this week either as europecommons.org is
> launching in next few days, but also could take a closer look soon.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/civic-commons/opencivic/issues/114#issuecomment-20042411
> .

## 

Adi Eyal
Data Specialist
phone: +27 78 014 2469
skype: adieyalcas
linkedin: http://za.linkedin.com/pub/dir/Adi/Eyal

**Comment by pmackay** [comment]
Yes, top down one at a time. Report if you have a similar problem.
